PROTO_tests/tests/client_dtypes_test.py

```diff
@@ -0,0 +1,183 @@
+import ipaddress
+import random
+
+import pytest
+
+import arkouda as ak
+
+# from arkouda import client_dtypes
+
+INT_TYPES = [ak.int64, ak.uint64]
+
+
+class TestClientDType:
```

Comment:
Remove unused import and update class name to match function (i.e. make it plural `ClientDTypes`)

```suggestion

INT_TYPES = [ak.int64, ak.uint64]


class TestClientDTypes:
```

Reply:
@jaketrookman you only did half of my suggestion. You didn't make the class name plural

Reply:
Updated for real this time